deps/curl-7.88.1/lib/vtls/sf_ocsp.c

```diff
@@ -758,6 +809,7 @@ static OCSP_RESPONSE * queryResponderUsingCurl(char *url, OCSP_CERTID *certid, c
 
 end:
   if (ocsp_req_base64) curl_free(ocsp_req_base64);
+  if (encoded_ocsp_req_base64) curl_free(encoded_ocsp_req_base64);
```

Comment:
You allocated using malloc and then free using curl_free. is that ok?

Reply:
This is following the existing curl implementation here:
https://github.com/snowflakedb/libsnowflakeclient/blob/master/deps/curl-7.88.1/lib/base64.c#L206
The buffer ocsp_req_base64 is allocated with malloc and freed using curl_free as well.